.babelrc

```diff
@@ -6,6 +6,7 @@
   "plugins": [
     ["module-resolver", {
       "root": ["./src", "."]
-    }]
+    }],
+    "transform-runtime"
```

Comment:
Thia should be taken care from the app that is going to implement and use this library if the sources want to be used